schema/test/unit/computed_columns/form_change_as_project_contact_test.sql

```diff
@@ -0,0 +1,51 @@
+
+
+begin;
+
+select plan(2);
+
+/** SETUP **/
+truncate table cif.change_status restart identity cascade;
+
+insert into cif.change_status (status, triggers_commit, active)
+values
+  ('pending', false, true);
+
+insert into cif.project_revision(id, change_status)
+  overriding system value
+  values (1, 'pending');
+
+insert into cif.form_change(id, operation, form_data_schema_name, form_data_table_name, form_data_record_id, project_revision_id, json_schema_name, new_form_data)
+  overriding system value
+  values
+    (1, 'create', 'cif', 'project_contact', 1, 1, 'project_contact', '{"projectId": 1, "contactId": 1, "contactIndex": 1}'),
+    (2, 'create', 'cif', 'operator', 1, 1, 'operator', '{"legalName": "I am an operator"}');
```

Comment:
Hm, right. The SQL function still works, but relay complains about the non-nullable rowId for that record. I guess we're only interested in the new_form_data and the id field here isn't really important, I can generate an id here instead of deriving it from form_data_record_id.

Reply:
I think I may do something like this:
`(select max(id) + 100 from cif.project_contact) as id`
This will always return an integer value for the id, but never point to an actual record. Using the form_change record's id will point to a different project_contact record than expected, which could be confusing to both developers and relay.

Reply:
Actually, now that I'm trying to use this in the diff, I see that having the same ID per form_change record is confusing relay, it thinks we're trying to overwrite data on the two project / project_contact records being queried. The ID should be the id of the form_change id passed in as a parameter

Reply:
Can we use a negative id to avoid overriding the real `project_contact` records? Not sure if they're being fetched anywhere right now though

Reply:
Negative ID's are a great idea